src/main/webapp/i18n/de/tutorExerciseDashboard.json

```diff
@@ -14,6 +14,8 @@
             "readGradingInstructions": "Bewertungsanweisungen lesen",
             "startYourParticipation": "Beginnen Sie Ihre Teilnahme an der Übung.",
             "gradingInstructions": "Bewertungsanweisungen",
+            "problemStatement": "Aufgabenstellung",
+            "exampleSolution": "Beispiel Lösung",
```

Comment:
`Lösungsbeispiel` or `Beispiellösung` would be a better wording

Reply:
I am fine with either, however we have to be consistent in the whole user interface.